runtime/onert/core/src/ir/LoweredGraph.cc

```diff
@@ -446,7 +446,7 @@ bool LoweredGraph::mergeable(const OpSequenceIndex &op_seq_index, const Operatio
       const auto &n = _graph.operations().at(n_index);
 
       // node's output == op_seq's input?
-      const auto &n_inputs = n.getInputs();
+      const auto &n_inputs = n.getInputs() | ir::Remove::UNDEFINED;
```

Comment:
How about moving this filter to in for()?

Reply:
@ragmani I like it. I've updated.